packages/expo/src/Notifications/Notifications.ts

```diff
@@ -438,4 +441,50 @@ export default {
     }
     return ExponentNotifications.setBadgeNumberAsync(number);
   },
+
+  async scheduleNotificationWithCalendarAsync(
+    notification: LocalNotification,
+    options: {
+      year?: number;
+      month?: number;
+      hour?: number;
+      day?: number;
+      minute?: number;
+      second?: number;
+      weekDay?: number;
+      repeat?: boolean;
+    } = {}
+  ): Promise<string> {
+    const areOptionsValid: boolean =
+      (options.month == null || isInRange(options.month, 1, 12)) &&
+      (options.day == null || isInRange(options.day, 1, 31)) &&
+      (options.hour == null || isInRange(options.hour, 0, 23)) &&
+      (options.minute == null || isInRange(options.minute, 0, 59)) &&
+      (options.second == null || isInRange(options.second, 0, 59)) &&
+      (options.weekDay == null || isInRange(options.weekDay, 1, 7)) &&
+      (options.weekDay == null || options.day == null);
+
+    if (!areOptionsValid) {
+      throw 'Options in scheduleNotificationWithCalendarAsync call were incorrect!';
+    }
+
+    return ExponentNotifications.scheduleNotificationWithCalendar(notification, options);
+  },
+
+  async scheduleNotificationWithTimerAsync(
+    notification: LocalNotification,
+    options: {
+      interval: number;
+      repeat?: boolean;
+    }
+  ): Promise<string> {
+    if (options.interval < 1) {
+      throw 'Interval must be not less then 1';
```

Comment:
Please throw Errors, not strings. 🙂 You could even use `CodedError` from `@unimodules/core`